ci/test/resources/hello.py

```diff
@@ -19,10 +20,12 @@ async def get_healthcheck(request):  # pylint: disable=W0613
 
 
 @routes.get('/sha')
-async def get_sha(request):
+async def get_sha(request):  # pylint: disable=W0613
```

Comment:
can we put the name of the error rather than the code. I think you want `# pylint: disable=unused-argument`